README.md

```diff
@@ -8,12 +8,37 @@
 [![MPL-2.0 License](https://img.shields.io/badge/license-MPL_2.0-blue.svg)](https://www.mozilla.org/en-US/MPL/2.0/)
 [![Join the chat at https://gitter.im/powsybl/powsybl-core](https://badges.gitter.im/powsybl/powsybl-core.svg)](https://gitter.im/powsybl/powsybl-core?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) 
 
-http://www.powsybl.com
+Powsybl (Power System Blocks) is an open source framework written in Java, that makes it easy to write complex software for power systems’ simulations and analysis.
+
+![PowSyBl Logo](https://raw.githubusercontent.com/powsybl/powsybl-gse/master/gse-spi/src/main/resources/images/logo_lfe_powsybl.svg?sanitize=true)
+
+Read more at http://www.powsybl.com !
+
+This project and everyone participating in it is governed by the [PowSyBl Code of Conduct](https://github.com/powsybl/.github/blob/master/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [powsybl.ddl@rte-france.com](mailto:powsybl.ddl@rte-france.com).
+
+## Project Structure
```

Comment:
Should we have a `Getting started` section to redirect people to the [itools](https://powsybl.github.io/docs/installation/itools-packager.html) and [javaFX](https://powsybl.github.io/docs/installation/javafx-packager.html) documentation pages before explaining how to build powsybl from sources?

Reply:
I reworded the intro and talked about downloading demos (which we have to build) and getting guidance on starting you own app with a link to doc/tutorials